plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -1238,11 +1319,17 @@ def validate_radio_series(self, ap_config):
         for radio_type in self.radio_interface:
             ap_series = ap_config.get(radio_type)
             self.log('Validating radio type: {0}'.format(radio_type), "INFO")
+
             if ap_series is not None:
                 for series in self.allowed_series[radio_type]:
-                    pattern = r'\b{}\w+'.format(re.escape(series))
-                    compiled_pattern = re.compile(pattern)
-                    is_valid = compiled_pattern.search(self.payload["access_point_details"]["series"])
+                    compiled_pattern = [re.compile(r'\b{}\w+'.format(re.escape(series))),
+                                        re.compile(r'\b{}\b'.format(re.escape(series)))]
+                    is_valid = compiled_pattern[0].search(self.payload["access_point_details"]["series"])
```

Comment:
Can't we write it in a single loop and make it simple.. If is_valid is true, then we can break otherwise append invalid_series..

```
    for i in range(2):
        is_valid = compiled_pattern[i].search(self.payload["access_point_details"]["series"])
        if is_valid:
            invalid_series = []
            break

        invalid_entry = "Access Point series '{0}' not supported for the radio type {1} allowed series {2}".format(
                            self.payload["access_point_details"]["series"],
                            radio_type,
                            str(self.allowed_series[radio_type])
                        )
..
..

                        
```

Reply:
it is not working, break is coming out from current for loop only  we need to come out one step back "series" for loop.

Reply:
i have changed regex and tested it is working. code not repeated.